pandas/tseries/tests/test_resample.py

```diff
@@ -1910,13 +1906,36 @@ def test_resample_empty(self):
         # GH12771
         index = PeriodIndex(start='2000', periods=0, freq='D', name='idx')
         s = Series(index=index)
-        result = s.resample('M').sum()
+        results = [s.resample('M').sum(), s.resample('M').count(),
```

Comment:
we don't do multiple tests at the same time, do them separately and one after the other